_includes/banner.html

```diff
@@ -1,4 +1,3 @@
-<span id="github-ribbon"><a href="https://github.com/swcarpentry/bc">Fork me on GitHub</a></span>
```

Comment:
there are 2 issues:
- from what I gathered, I think the github ribbon shouldn't be included inside the banner as the new bootstrap default messes up the positioning of the ribbon (see swcarpentry/styles#4) and therefore I moved it to workshop.html for now (side effect it won't show up on other pages, but I'll fix that once I figure out the CSS issue)
- the URL is not appropriate anymore because of the reorganization of the repositories, correctly addressed by #148